pkg/controller/datavolume/controller-base.go

```diff
@@ -1030,7 +1017,11 @@ func (r *ReconcilerBase) addOwnerRef(pvc *corev1.PersistentVolumeClaim, dv *cdiv
 }
 
 func updateProgressUsingPod(dataVolumeCopy *cdiv1.DataVolume, pod *corev1.Pod) error {
-	httpClient = cc.BuildHTTPClient(httpClient)
+	// Create a new http.Client
+	defaultClient := &http.Client{}
+
+	// Pass the http.Client to BuildHTTPClient function
+	httpClient := cc.BuildHTTPClient(defaultClient)
```

Comment:
you probably want to pass a nil to BuildHTTPClient so you get a fresh one, otherwise, 
you are getting the same non-nil client that you passed `http.Client{}` which is probably not what you want

Reply:
I will revert my changes, I changes it only since the test failed on it, but maybe something changed

Reply:
I have a bug here sorry about that fixing